products/workers/src/content/platform/pricing.md

```diff
@@ -5,98 +5,152 @@ pcx-content-type: concept
+| -------------------- | ---------------------------- | ------------------------------------------------------------ | ---------------------------- |
+| Requests             | 100,000 / day                | 1 million, + $0.15/million                                   | 10 million, +$0.50/million   |
+| Duration             | 10ms CPU time / invocation   | 400,000 GB-s, + $12.50/million GB-s<sup>1</sup>,<sup>2</sup> | 50 ms CPU time / invocation  |
+  
+</TableWrap>
+
+1. Cloudflare will bill for duration charges based on the higher of your wall time or CPU time, with a multiple applied to the CPU time to account for the processing power allotted to your script. We will not bill for wall time Duration charges beyond the execution [limit](/platform/limits#worker-limits) given.
+2. Duration billing charges for the 128 MB of memory your Worker is allocated, regardless of actual usage.  If your account has significant traffic to a single Worker, multiple instance of that Worker may run in same isolate on the same physical machine and share the 128MB of memory. These Workers are still billed as if they are allocated a full 128MB of memory.
 
 ### Usage Models
 
-Under the Paid plan, you have access to two Usage Models: Bundled and Unbound. Usage Models are **settings on your Workers** that specify the upper [limits](/platform/limits) for how long a Worker can execute. In addition to different limits, Workers on the Bundled Usage Model have usage billing based on requests only, while Workers on Unbound have usage billing based on requests, duration, and egress data at the rates shown under [pricing](/platform/pricing#pricing).
+Workers are available under two Usage Models: Bundled and Unbound. Usage Models are **settings on your Workers** that specify the upper [limits](/platform/limits) for how long a Worker can execute. In addition to different limits, Workers on the Bundled Usage Model have usage billing based on requests only, while Workers on Unbound have usage billing based on requests and duration at the rates shown under [pricing](/platform/pricing#pricing).
 
-### Default Usage Model
+#### Default Usage Model
 
 The default Usage Model is used when new Workers are created, and no Usage Model is provided via configuration during creation. When an account is first upgraded to the Paid plan, the default Usage Model is set to Unbound. The default can be changed at any time by using the Default Usage Model setting on the Workers overview page. We recommend setting the default to the type of Worker you create the most. Existing Workers will not be impacted when changing the default Usage Model.
 
+### Same features
+
+Script size, number of scripts, subrequests, and available memory are not affected by plan type.
+
+### Workers billing examples
+
+#### Example 1
+
+If an Unbound Worker executed 1.5 million times and used a total of 200,000 GB-s, the estimated cost in a month would be:
+
+Total = ~$0.8 USD + Minimum $5/mo usage = **$5.08**
```

Comment:
first number should be 8 cents not 80 cents right?

Reply:
Agree,

```suggestion
Total = ~$0.08 USD + Minimum $5/mo usage = **$5.08**
```